src/modules/claims/claims.service.ts

```diff
@@ -1482,11 +1483,20 @@ export class ClaimsService {
     }
     const role = vc.credentialSubject.role.namespace;
     try {
+      let issuerVerificationResult: VerificationResult;
       if (typeof issuerDID === 'string') {
-        await this._issuerVerification.verifyIssuer(issuerDID, role);
+        issuerVerificationResult = await this._issuerVerification.verifyIssuer(
+          issuerDID,
+          role
+        );
       } else {
-        await this._issuerVerification.verifyIssuer(issuerDID.id, role);
+        issuerVerificationResult = await this._issuerVerification.verifyIssuer(
+          issuerDID.id,
+          role
+        );
       }
+      issuerVerified = issuerVerified && issuerVerificationResult.verified;
```

Comment:
`issuerVerified` is also used as result of status verification, so the final `issuerVerified` is boolean `AND` of both checks. Perhaps this checks should be separated